File: src/main/java/strategy/MyStrategy.java
package strategy;

import ai.AI;
import model.*;

public final class MyStrategy implements Strategy {

  private AI ai = new AI();
  @Override
  public void move(Wizard self, World world, Game game, Move move) {
    ai.updateWorldInfo(self, world, game, move);
    ai.makeDecision();
  }
}

File: src/main/java/MyStrategy.kt
import java.util.*;
import java.lang.*;
import model.*;

class MyStrategy : Strategy {val ai = AI()
override fun move(self: Wizard, world: World, game: Game, move: Move) { ai.updateInfo(self, world, game, move); ai.decision();}}

class AI {

  fun decision() {

  }

  fun updateInfo(self: Wizard, world: World, game: Game, move: Move) {

  }

}


